release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -0,0 +1,1663 @@
+|Supported
+|====
+
+.CSI Volumes
+[cols="2,1,1",options="header"]
+|====
+|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
+
+|Cloning
+|Supported
+|Supported
+
+|Expansion
+|Supported
+|Supported
+
+|Snapshot
+|Supported
+|Supported
+|====
+
+[id="ocp-4-17-auth_{context}"]
+=== Authentication and authorization
+
+[id="ocp-4-17-networking_{context}"]
+=== Networking
+
+[id="ocp-4-17-registry"]
+=== Registry
+
+[id="ocp-4-17-storage_{context}"]
+=== Storage
+
+[id="ocp-4-17-oci"]
+=== {oci-first}
+
+[id="ocp-4-17-oci_{context}"]
+=== Oracle(R) Cloud Infrastructure
+
+[id="ocp-4-17-olm_{context}"]
+=== Operator lifecycle
+
+[id="ocp-4-17-osdk_{context}"]
+=== Operator development
+
+[id="ocp-4-17-builds_{context}"]
+=== Builds
+
+[id="ocp-4-17-machine-config-operator_{context}"]
+=== Machine Config Operator
+
+[id="ocp-4-17-machine-management_{context}"]
+=== Machine management
+
+[id="ocp-4-17-nodes_{context}"]
+=== Nodes
+
+[id="ocp-4-17-monitoring_{context}"]
+=== Monitoring
+
+[id="ocp-4-17-network-observability-1-6_{context}"]
+=== Network Observability Operator
+The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, Rolling Stream which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator is found in the xref:../observability/network_observability/network-observability-operator-release-notes.adoc#network-observability-rn[Network Observability release notes].
+
+[id="ocp-4-17-scalability-and-performance_{context}"]
+=== Scalability and performance
+
+[id="ocp-4-17-edge-computing_{context}"]
+=== Edge computing
+
+[id="ocp-4-17-hcp_{context}"]
+=== Hosted control planes
+
+[id="ocp-4-17-insights-operator_{context}"]
+=== Insights Operator
+
+[id="ocp-4-17-etcd-certificates_{context}"]
+=== Security
+
+[id="ocp-4-16-notable-technical-changes_{context}"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+
+// Note: use [discrete] for these sub-headings.
+// example sub-heading below:
+//[discrete]
+//[id="ocp-4-16-cluster-cloud-controller-manager-operator"]
+//=== Cloud controller managers for additional cloud providers
+//
+
+[id="ocp-4-16-deprecated-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
```

Comment:
```suggestion
Deprecated functionality is still included in {product-title} and continues to be supported. However, it will be removed in a future release of this product. Avoid using the deprecated functionality for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
```
As per RH SSG, avoid the verb recommend. Refer to https://redhat-documentation.github.io/supplementary-style-guide/#recommend

Reply:
This has been the phrasing in the release notes for a long time so I'm going to leave it for consistency.